extensions/wikia/Forum/ForumBoardInfo.class.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+class ForumBoardInfo {
+
+	private $id;
+	private $name;
+	private $url;
+	private $description;
+	private $postCount;
+	private $threadCount;
+	/** @var ForumPostInfo $lastPost */
+	private $lastPost;
+
+	/**
+	 * ForumBoardInfo constructor.
+	 * @param array $array ;
+	 */
+	public function __construct( $array = [ ] ) {
+		$this->setLastPost( new ForumPostInfo( $array[ 'lastPost' ] ) );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "lastPost" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)